content/docs/command-reference/gc.md

```diff
@@ -62,7 +62,8 @@ The default remote is cleaned (see `dvc config core.remote`) unless the
   versions of an experiment or project. Note that both options can be combined,
   for example using the `-aT` flag.
 
-- `--all-commits` - same as `-a` or `-T` above, but applies to all Git commits,
+- `--all-commits` - same as `-a` or `-T` above, but applies to all
+  [reachable](https://git-scm.com/docs/gitglossary#def_reachable) Git commits,
```

Comment:
Included this one too since we emphasize from Exp docs that `exp gc` only does half of the job.